sirepo/uri_router.py

```diff
@@ -257,6 +257,10 @@ def _init_uris(app, simulation_db):
     assert _default_route, \
         'missing default route'
     _empty_route = _uri_to_route.en
+    i = set(_uri_to_route.keys()).union(sim_types).intersection(
```

Comment:
Nice. Need the intersection of _uri_to_route with sim_types, just as a safety feature so this might be easiest:

```py
u = set(_uri_to_route.keys())
r = simulation_db.SCHEMA_COMMON.rootRedirectUri.keys()
t = sim_types
i = u & r | u & t | r & t
assert not i,rootRedirectUri, sim_types, and routes have overlapping uris={i}'
```

Reply:
https://github.com/radiasoft/sirepo/pull/2786/commits/1912882dc322db1f33962e87148aeddc9c6ef2d7